tuxemon/event/actions/teach.py

```diff
@@ -0,0 +1,74 @@
+#
+# Tuxemon
+# Copyright (c) 2020      William Edwards <shadowapex@gmail.com>,
+#                         Benjamin Bean <superman2k5@gmail.com>
+#
+# This file is part of Tuxemon
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Contributor(s):
+#
+# Adam Chevalier <chevalierAdam2@gmail.com>
+
+from __future__ import annotations
+
+import logging
+import uuid
+from typing import NamedTuple, final
+
+from tuxemon.event.eventaction import EventAction
+from tuxemon.technique.technique import Technique
+
+logger = logging.getLogger(__name__)
+
+
+class TeachActionParameters(NamedTuple):
+    monster_id: str
+    technique: str
+
+
+# noinspection PyAttributeOutsideInit
```

Comment:
?

Reply:
It was part of the file I utilized. I'm going to remove it.

This string there is also in:
clear_variable
copy_variable
get_player_monster
spawn_monster
store_monster

Do you want that I open another PR to remove it from all?